models.py

```diff
@@ -208,3 +204,130 @@ def get_data_for_display():
         is_everything_ok = servers_down_count == 0
 
         return (is_everything_ok, servers_statuses)
+
+
+class VariableType(Enum):
+    INTEGER = 'INTEGER'
+    FLOAT = 'FLOAT'
+    STRING = 'STRING'
+    BOOL = 'BOOL'
+    ARROW = 'ARROW'
+
+
+class Variable(db.Model):
+    __tablename__ = 'variables'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+
+    name = db.Column(db.String(255), nullable=False, unique=True)
+    type = db.Column(db.Enum(VariableType), nullable=False)
+    _value = db.Column('value', db.String)
+
+    @property
+    def value(self):
+        """Get the real (cast) value of this Variable."""
+        if self._value and self.type != VariableType.STRING: # No need to cast STRING values
+            if self.type == VariableType.INTEGER:
+                return int(self._value)
+            elif self.type == VariableType.FLOAT:
+                return float(self._value)
+            elif self.type == VariableType.BOOL:
+                return bool(int(self._value))
+            elif self.type == VariableType.ARROW:
+                return arrow.get(self._value)
+            else:
+                raise ValueError('Unhandled value type')
+
+        return self._value
+
+    @value.setter
+    def value(self, value):
+        """Set the value of this Variable."""
+        if isinstance(value, int):
```

Comment:
Refactor those if else and use one if else for the value and another for the type